### PR TITLE
[NNAPI EP] Add support for PRelu op

### PR DESCRIPTION
**Pull request by edgchen1**
**Description**
Add NNAPI EP support for PRelu op.

**Motivation and Context**
PRelu use in a model.

### CODE REVIEW

**Comment by skottmckay** [comment]
Any additional unit tests required or there are existing PRelu test that hit the changes?

File: onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc [hunk 1]
```diff
@@ -620,7 +621,10 @@ Status BinaryOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const
   } else if (op_type == "Pow") {
     add_activation = false;  // ANEURALNETWORKS_POW does not have activation
     op_code = ANEURALNETWORKS_POW;
-  } else {
+  } else if (op_type == "PRelu") {
+    add_activation = false;  // ANEURALNETWORKS_PRELU does not have activation
+    op_code = ANEURALNETWORKS_PRELU;
+    } else {
```
Comment (YUNQIUGUO) [inline onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc:627]: not sure about this but - Does the second input of PRelu op, `slope` needs any special handling as an initializer? or can it just share the binaryopbuilder

Comment (skottmckay) [reply onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc:627]: The NNAPI [spec](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a82e7e7c709ad117a0178203217be30cf) doesn't require any special scaling etc. of the slope input, and allows the scale and zp to differ from the first input. Based on that, there's no processing we need to do ahead of time, so I don't think it should matter whether slope comes from an initializer or not.

Comment (YUNQIUGUO) [reply onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc:627]: ah ok - I see now. this was a question actually I was wondering before for initializers. thx for the explanation.

**Comment by edgchen1** [comment]
> Any additional unit tests required or there are existing PRelu test that hit the changes?

There are existing PRelu op tests: `ActivationOpTest.PRelu*`.
And the NNAPI EP is enabled for op tests in the Android CI build.

https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=633710&view=logs&j=647f6050-e2ec-5ab7-0818-bbad84ce0ace&t=3ef45123-eafb-5911-7976-bbcc686d3bfe

Let me know if there any other suggested tests to add.

**Review by skottmckay — APPROVED** [review]
:shipit:

**Pull request merged**